.github/dependabot.yml

```diff
@@ -17,6 +17,8 @@ updates:
     - dependency-name: "wasm-bindgen"
     - dependency-name: "once_cell"
     - dependency-name: "chrono"
+    - dependency-name: "js-sys"
+    - dependency-name: "web-sys"
```

Comment:
Are these off-target? I'm guessing it's because they generate complications downstream on the plugins when updated?

Reply:
yes, updating these prevented installing with the branchwater plugin :/